Farm cards new UI expanded

memacius32 opened the pull request:
Adding Expanded section content:

- Claimable Rewards.
- User deposits.
- Pool Type and Total staked (in mobile)
- Manage farm box

Note: 
Will Remove Manage page in a new pr (as it's now unused)

### Preview: 
Desktop:
<img width="715" alt="Screen Shot 2022-09-14 at 01 06 48" src="https://user-images.githubusercontent.com/96993065/190024673-0f07fdac-542c-41ee-b83c-9c3c8d1d65ee.png">
Mobile:
<img width="386" alt="Screen Shot 2022-09-14 at 01 06 04" src="https://user-images.githubusercontent.com/96993065/190024663-f7a17b43-b826-4a5c-a8df-10466845c90d.png">

#### Video:

https://user-images.githubusercontent.com/96993065/190025315-ef5ab212-9826-410c-9558-def10955db1b.mp4

specialist0x commented:
Can you address the `50%` `max` buttons, it looks like one is selected when it shouldn't be:

<img width="362" alt="image" src="https://user-images.githubusercontent.com/94581898/191376805-0cf7050b-a7e0-4321-b180-c8f6ac6787b4.png">

specialist0x commented on src/components/Toggle/index.tsx:
```diff
@@ -3,15 +3,15 @@ import styled from 'styled-components'
   font-weight: 400;
 
-  padding: 0.35rem 0.6rem;
+  padding: ${({ padding }) => (padding ? padding : '0.35rem 0.6rem')};
```
```suggestion
  padding: ${({ padding }) => (padding ?? '0.35rem 0.6rem')};
```

specialist0x commented on src/components/earn/PoolCardTri/ManageStake.tsx:
```diff
@@ -0,0 +1,172 @@
+    event.stopPropagation()
+  }
+
+  function handleWithDrawClick(event: React.MouseEvent) {
```
```suggestion
  function handleWithdrawClick(event: React.MouseEvent) {
```

specialist0x commented on src/components/earn/PoolCardTri/PoolCardTri.styles.tsx:
```diff
@@ -103,26 +108,30 @@ export const StyledPairContainer = styled(PairContainer)`
+  `};
   ${({ theme }) => theme.mediaWidth.upToExtraSmall`
-    grid-template-columns: 180px  auto 85px;
+    grid-template-columns: 167.5px  auto 92px;
```
I don't think you can have pixels as decimals

specialist0x commented on src/components/earn/PoolCardTri/index.tsx:
```diff
@@ -93,18 +116,18 @@ const DefaultPoolCardtri = ({
 
   function handleClaimClick(event: React.MouseEvent) {
-    enableModal()
+    enableClaimModal()
```
Should `event.stopPropagation()` be invoked first?  Might cause a race condition

memacius32 replied:
Changed!

specialist0x reviewed (approved):
Accepting to unblock, please make suggested changes

memacius32 commented:
> Can you address the `50%` `max` buttons, it looks like one is selected when it shouldn't be:
> 
> <img alt="image" width="362" src="https://user-images.githubusercontent.com/94581898/191376805-0cf7050b-a7e0-4321-b180-c8f6ac6787b4.png">

Good catch, fixed 🙏 
<img width="422" alt="Screen Shot 2022-09-21 at 15 44 40" src="https://user-images.githubusercontent.com/96993065/191520519-6f1177f3-da0b-41bc-a62f-c9f1fb94ab20.png">

Addressed rest of coments and Merging
